js/src/components/notices.js

```diff
@@ -3,30 +3,43 @@ const Notices = {
 	_init() {
 		const self = this;
 		if ( typeof CLDIS !== 'undefined' ) {
-			const notices = document.getElementsByClassName( 'cld-notice' );
+			const notices = document.getElementsByClassName( 'cld-notice-box' );
 			[ ...notices ].forEach( ( notice ) => {
-				notice.addEventListener( 'click', ( ev ) => {
-					// WordPress has an onClick that cannot be unbound.
-					// So, we have the click on our Notice, and act on the
-					// button as a target.
-					if ( 'notice-dismiss' === ev.target.className ) {
-						self._dismiss( notice );
-					}
-				} );
+				const dismiss = notice.getElementsByClassName(
```

Comment:
@DavidCramer 

we could had used here as well `.querySelector`, and that should return you only 1 or 0 items. It wouldn't require de use of `[0]` ahead.